rcsb/db/processors/DataTransformFactory.py

```diff
@@ -227,7 +230,13 @@ def processRecord(self, tableId, row, attributeNameList, containerName=None):
                         if dT["pureCast"][atName] == "string":
                             dD[dT["atNameD"][atName]] = row[ii]
                         elif dT["pureCast"][atName] == "integer":
-                            dD[dT["atNameD"][atName]] = int(row[ii])
+                            if int(row[ii]) > 2147483647 and self.__transFlags["dropLargeIntegers"]:
```

Comment:
I think this needs to check the absolute value, since it also applies to negative integers